src/components/user-name.jsx

```diff
@@ -1,33 +1,32 @@
 import React from 'react'
-import {Link} from 'react-router'
-import {connect} from 'react-redux'
-import {preventDefault} from '../utils'
+import { Link } from 'react-router'
+import { connect } from 'react-redux'
+import { preventDefault } from '../utils'
 
 const DisplayUsername = ({mode, user}) => {
-  switch(mode){
-    case 'screen':{
+  switch (mode) {
+    case 'screen':
+      {
       return <span>{user.screenName}</span>
-    }
-    case 'user':{
+      }
+    case 'user':
+      {
       return <span>{user.username}</span>
-    }
+      }
   }
 
   return <span>{`${user.screenName} `}<span className='be-fe-username'>{user.username}</span></span>
 }
 
 const UserName = (props) => (
-  <Link
-    className={`user-name-info ${props.className}`}
-    to={`/${props.user.username}`}
-  >
-    {props.display
-      ? <span>{props.display}</span>
-      : <DisplayUsername mode={props.settings.userNameMode} user={props.user}/>}
-  </Link>
+<Link className={`user-name-info ${props.className}`} to={`/${props.user.username}`}>
+{props.display
+   ? <span>{props.display}</span>
+   : <DisplayUsername mode={props.settings.userNameMode} user={props.user} />}
+</Link>
```

Comment:
i9n.

But I'd like to merge my PR first, because there's rewritten `user-name.jsx`.